integration-tests/testsuite/src/test/java/io/apicurio/tests/multitenancy/MultitenantNoAuthIT.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.tests.multitenancy;
+
+import io.apicurio.multitenant.api.beans.TenantStatusValue;
+import io.apicurio.multitenant.api.datamodel.NewRegistryTenantRequest;
+import io.apicurio.multitenant.api.datamodel.RegistryTenant;
+import io.apicurio.multitenant.api.datamodel.UpdateRegistryTenantRequest;
+import io.apicurio.multitenant.client.TenantManagerClient;
+import io.apicurio.multitenant.client.TenantManagerClientImpl;
+import io.apicurio.registry.rest.client.RegistryClient;
+import io.apicurio.registry.rest.client.RegistryClientFactory;
+import io.apicurio.registry.rest.client.exception.RestClientException;
+import io.apicurio.registry.rest.v2.beans.ArtifactMetaData;
+import io.apicurio.registry.types.ArtifactType;
+import io.apicurio.registry.utils.tests.TestUtils;
+import io.apicurio.tests.common.ApicurioRegistryBaseIT;
+import io.apicurio.tests.common.Constants;
+import io.apicurio.tests.common.RegistryFacade;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/**
+ * @author Fabian Martinez
+ * @author Jakub Senko <jsenko@redhat.com>
+ */
+@Tag(Constants.MULTITENANCY)
+public class MultitenantNoAuthIT extends ApicurioRegistryBaseIT {
```

Comment:
I suggest to rename this class to something specific to the reaper process, or tenant deprovisioning ...
Multitenancy integration tests will all be forced to use authentication soon, so this name won't make sense in the near future.

Reply:
Ok